packages/ui/test/forum/components/WatchlistThreadsBrowser.test.tsx

```diff
@@ -14,6 +14,9 @@ import { setupMockServer } from '../../_mocks/server'
 describe('UI: WatchlistThreadsBrowser', () => {
   const server = setupMockServer({ noCleanupAfterEach: true })
 
+  const matchHeader = (regex: RegExp) => (content: string, element: Element | null) =>
+    element?.tagName.toLowerCase() == 'h5' && regex.test(content)
```

Comment:
Did you saw the [`selector` option for `findByText()`](https://testing-library.com/docs/queries/bytext#selector)? Shouldn't that do the trick?

Reply:
It does, must've missed it